compress.cc

```diff
@@ -113,6 +108,22 @@ compression_parameters::compression_parameters(compressor_ptr c)
     : _compressor(std::move(c))
 {}
 
+std::string_view compression_parameters::algorithm_to_name(algorithm alg) {
+    switch (alg) {
+        case algorithm::lz4: return "LZ4Compressor";
+        case algorithm::deflate: return "DeflateCompressor";
+        case algorithm::snappy: return "SnappyCompressor";
+        case algorithm::zstd: return "ZstdCompressor";
+        case algorithm::none: abort();
```

Comment:
It's too aggressive to kill a server because of some bug that no one noticed, better on_internal_error() (that only throws in production).

Reply:
Done.